web/src/app/services/AlertMetrics/AlertMetricsCSV.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import { Grid, Typography, Button } from '@mui/material'
+import DownloadIcon from '@mui/icons-material/Download'
+import makeStyles from '@mui/styles/makeStyles'
+import { CSVLink } from 'react-csv'
+import { Alert } from '../../../schema'
+import { theme } from '../../mui'
+import { DateTime } from 'luxon'
+
+interface AlertMetricsCSVProps {
+  alerts: Alert[]
+}
+
+const useStyles = makeStyles({
+  paragraph: {
+    display: 'flex',
+    justifyContent: 'flex-end',
+  },
+  anchor: {
+    color: theme.palette.primary.main,
+    '&:hover': {
+      textDecoration: 'none',
+    },
+  },
+})
+
+export default function AlertMetricsCSV(
+  props: AlertMetricsCSVProps,
+): JSX.Element {
+  const classes = useStyles()
+  const zoneAbbr = DateTime.local().toFormat('ZZZZ')
+  // Note: the data object is ordered
+  const data = props.alerts.map((a) => ({
+    [`createdAt (${zoneAbbr})`]: DateTime.fromISO(a.createdAt).toLocal().toSQL({
```

Comment:
So this one is interesting, it shows the full value in when i click on a cell but just the `minutes:seconds.milliseconds` in the cell preview. Is there a way around that?

<img width="315" alt="Screen Shot 2022-01-17 at 11 38 37 AM" src="https://user-images.githubusercontent.com/11381794/149838036-792d9197-295a-4b0a-a8a6-8098ef1cbed6.png">